packages/amplication-client/src/Resource/ResourceHome.tsx

```diff
@@ -50,7 +114,60 @@ const ResourceHome = ({ match, innerRoutes }: Props) => {
                 resourceThemeMap[currentResource?.resourceType].color,
             }}
           >
-            {currentResource?.name}
+            <div className={`${CLASS_NAME}__header__input__container`}>
```

Comment:
pelase use a seperate component for this form

Reply:
Sure, will work on it